docs/import-data/files/cypherl.md

```diff
@@ -1,35 +1,62 @@
 ---
 id: cypherl
 title: Importing Cypher queries (.cypherl format)
-sidebar_label: Cypher queries (.cypherl format)
+sidebar_label: .cypherl
 pagination_prev: import-data/overview
 ---
 import Tabs from "@theme/Tabs"; import TabItem from "@theme/TabItem";
 
-When Memgraph is running, Cypher queries are imported by running
-[mgconsole](/connect-to-memgraph/mgconsole.md) in non-interactive mode and
-importing data saved in a `.cypherl` file. 
+If your data is in the form of Cypher queries (`CREATE` and `MERGE` clauses)
+within a **.cypherl** file it can be imported via Memgraph
```

Comment:
bcs JSON and CSV are not extensions... like .json and .csv... The name of the format is TXT but the extension is `.txt` ...
That is the reason I code .cypherl
Maybe we can call them CYPHERL files?

Reply:
If this is an extension we came up with, then I would say it's totally fine to call the format CYPHERL (looks nicer too).

Reply:
OK, will change EVERYWHERE!

Reply:
Btw just to help out why the file with Cypher per line is called `.cypherl` because it is not the standard. Well, it is using the standard for the naming which is: `CYPHER` + `LINE` (per line) = `cypherl`.

The same applies to JSON, if it is a single JSON in the file it is `.json`. If it is JSON per line, it should be named `.jsonl` - Reference: https://jsonlines.org/.